server/src/main/java/org/elasticsearch/rest/action/document/RestBulkAction.java

```diff
@@ -82,6 +82,10 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         Boolean defaultRequireAlias = request.paramAsBoolean(DocWriteRequest.REQUIRE_ALIAS, null);
         bulkRequest.timeout(request.paramAsTime("timeout", BulkShardRequest.DEFAULT_TIMEOUT));
         bulkRequest.setRefreshPolicy(request.param("refresh"));
+        if (request.hasParam("no_items_on_success")) {
```

Comment:
Maybe `empty_response_on_success` is a better name?

Reply:
`response` refers to the whole json object.There is still a json response returned but with items empty. 

`empty_items_on_success` a happy medium?

Reply:
After a second thought maybe keep the name how it currently is.